### DISCUSSION TITLE
How to dynamically create element

### DISCUSSION THREAD

**Discussion by Cweili** [solidjs/solid#639] [Q&A] [answered]
I tried [this](https://playground.solidjs.com/?hash=1397771162&version=1.1.1), but failed with an error `TypeError: Comp is not a function`.

```jsx
import { render } from "solid-js/web";

function Element() {
  const DynamicElement = 'div'

  return (
    <DynamicElement>
      content
    </DynamicElement>
  );
}

render(() => <Element />, document.getElementById("app"));
```

**Comment by galchenkov** [comment]
May be 
```tsx
const DynamicElement = (props) => <div>{props.children}</div>
```

**Comment by atk** [comment, marked as answer]
Have a look at <Dynamic> https://www.solidjs.com/docs/latest/api#%3Cdynamic%3E – it does exactly what you want.

**Reply by Cweili** [reply]
Thanks! I just realized that it doesn’t just support components.
